package/appimage.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -x
+
+ORIGIN_DIR=$(pwd)
+
+# Get our directory
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+appimg_dir="${DIR}/appimage"
+gem_path=$(ls "${DIR}"/vagrant*.gem)
```

Comment:
This currently doesn't exist in the package dir....should it?

Reply:
no